src/Identifier/IdentifierInterface.php

```diff
@@ -18,10 +18,10 @@
     /**
      * Identifies an user or service by the passed credentials
      *
-     * @param mixed $credentials Authentication credentials
```

Comment:
Should be \ArrayAccess|null. You have probably overwritten changes to the master.
Try git rebase upstream/master so the changes are reflected in the correct order.